### PR TITLE
Update 00020-SupportingFunctionalities.md

### PR DESCRIPTION
**Pull request by rekjuh**
Suggestion 1 for this file. Note: I suggest the name to be changed "Compatibility and support"

### CODE REVIEW

File: 5 Other functionalities/00020-SupportingFunctionalities.md [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
+## Compatibility and support
 
-Some supporting features included are listed in the following paragraphs.
+This section outlines the various languages, browsers etc. Oskari supports, along with other compability considerations.
 
-### Multiple language support
+### Multilingual support
 
-Oskari fully supports English, Finnish and Swedish languages, and partially languages listed below.
+Oskari offers full support for English, Finnish and Swedish, and partially support for the following languages:
```
Comment (ZakarFin) [inline 5 Other functionalities/00020-SupportingFunctionalities.md:7]: Maybe something like Oskari supports localization of the software and currently has translations for... ?

Comment (rekjuh) [reply 5 Other functionalities/00020-SupportingFunctionalities.md:7]: Yes! 👍

File: 5 Other functionalities/00020-SupportingFunctionalities.md [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
 - Chrome
+- Firefox
+- Microsoft Edge
 - Safari
 
-### Multiple screen support
+### Multi-screen support
 
-Oskari interface works with tablets and mobile, but there's currently no separate mobile or tablet optimized version of Oskari. Embedded map tools are gradually developed into responsive tools which work well with smaller screens and are touch-friendly
+The Oskari interface is accessible on mobile device, including smartphones and tablets. However, there is currently **no** dedicated mobile or tablet-optimized version of Oskari (such as a mobile app). Embedded map tools are gradually being developed into responsive tools which work well with touchpad screens on mobile devices.
```
Comment (ZakarFin) [inline 5 Other functionalities/00020-SupportingFunctionalities.md:32]: Isn't touchpad something that's on laptops usually? Mobile devices just have touch screens? Embedded maps have supported these for fairly long time already. It's new that the geoportals work with mobile better now. So currently the geoportal is being developed to work better on smaller screens.

File: 5 Other functionalities/00020-SupportingFunctionalities.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+### Role-based user management
 
-Some Oskari modules can make use of role-based user management. Users belonging to different roles can be given access to specific functionality.
+Some Oskari modules utilize role-based user management, allowing users with different roles to access specific functionalities. For example, only logged-in users can view certain layers or spesific user groups can add or edit map layers within the service.
 
 ### SSO authentication support
 
-Authentication and SSO is supported via SAML 2.0.
-
+Oskari supports authentication and Single Sign-On (SSO) via SAML 2.0.
```
Comment (ZakarFin) [inline 5 Other functionalities/00020-SupportingFunctionalities.md:40]: Built-in SAML support has been removed. Oskari uses Spring security framework that can be customized to different use cases.

**Comment by rekjuh** [comment]
Done in PR 27.

**Pull request closed without merging**